src/pudl/metadata/codes.py

```diff
@@ -1115,4 +1134,57 @@
         "code_fixes": {},
         "ignored_codes": [],
     },
+    "steam_plant_types_eia": {
+        "df": pd.DataFrame(
+            columns=["code", "label", "description"],
+            data=[
+                (
+                    1,
+                    "steam_over_100mw",
+                    "Plants with combustible-fueled steam-electric generators with a sum of 100 MW or more steam-electric nameplate capacity (including combined cycle steam-electric generators with duct firing).",
+                ),
+                (
+                    2,
+                    "steam_between_10mw_and_100mw",
+                    "Plants with combustible-fueled steam-electric generators with a sum of 10 MW or more but less than 100 MW steam-electric nameplate capacity (including combined cycle steam-electric generators with duct firing).",
+                ),
+                (
+                    3,
+                    "nuclear_over_100mw",
+                    "Plants with nuclear fueled generators, combined cycle steam-electric generators without duct firing and solar thermal electric generators using a steam cycle with a sum of 100 MW or more steam-electric nameplate capacity.",
+                ),
+                (
+                    4,
+                    "non_steam",
+                    "Plants with non-steam fueled electric generators (wind, PV, geothermal, fuel cell, combustion turbines, IC engines, etc.) and electric generators not meeting conditions of categories above.",
+                ),
+            ],
+        ).convert_dtypes(),
+        "code_fixes": {},
+        "ignored_codes": [],
+    },
+    "reporting_frequencies_eia": {
+        "df": pd.DataFrame(
+            columns=["code", "label", "description"],
+            data=[
+                (
+                    "A",
+                    "annual",
+                    "The respondent only provides an annual total(s) for this record via the EIA-923 annual survey form.  Any monthly data in this record is estimated based on the respondent's reported annual total(s) and power plants with similar characteristics to this plant.",
+                ),
+                (
+                    "M",
+                    "monthly",
+                    "The respondent provides monthly values for this record and does so via the EIA-923 monthly survey form.",
+                ),
+                (
+                    "AM",
+                    "monthly_annual",
+                    "The respondent provides monthly values for this record, but does so once per year via the EIA-923 annual survey form.",
+                ),
+            ],
+        ).convert_dtypes(),
+        "code_fixes": {},
```

Comment:
Crazy that there are no codes to fix? Really?

Reply:
I know! wild!